Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 default-members = ["kube"]
+resolver = "2"
```

Comment:
i would remove this unless you had a reason for it, i vaguely remember it caused problems in the past.